policy/modules/roles/unconfineduser.te

```diff
@@ -158,6 +158,10 @@ optional_policy(`
         kpatch_run(unconfined_t,unconfined_r)
     ')
 
+	optional_policy(`
+		kerberos_tmp_filetrans_host_rcache(unconfined_t, "krb5_0.rcache2")
+	')
+
```

Comment:
Maybe you should rather add the "krb5_0.rcache2" entry to `kerberos_filetrans_named_content()` so that all unconfined domains get this transition (a lot of domains then get this transition, but it could avoid some further duplication of the magic filename in the future).

Reply:
Yes, sure :)